### DISCUSSION TITLE
Docs Request: Logsource

### DISCUSSION THREAD

**Discussion by defensivedepth** [SigmaHQ/sigma-specification#50] [Ideas]
I would suggest documenting the logsource for this rule: https://github.com/SigmaHQ/sigma-specification/blob/main/Taxonomy_specification.md#windows-folder

builtin/Mimikatz

REF: https://github.com/SigmaHQ/sigma/issues/3871#issuecomment-1372726896

**Comment by nasbench** [comment]
Should be resolved in #55 

I added a small description when using only the `product` as a log source that indicates `all logs of a specific OS`

**Comment by defensivedepth** [comment]
Thanks @nasbench 

For clarity, is the intent that `All Windows Logs` means all of the logsources defined under the Windows section there?

![image](https://user-images.githubusercontent.com/954732/213513337-a70b4534-753f-4b6b-9931-46500c6bdede.png)

**Reply by nasbench** [reply]
It's supposed to mean all windows logs that the person collects in their env in theory. Since the service/category specifies a set of channels or EIDs.

If you could think of better wording/description for it. I would add it.
